input/text/src/commonMain/kotlin/majestic/InputTextField.kt

```diff
@@ -1,117 +1,78 @@
 package majestic
 
 import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.foundation.text.KeyboardOptions
 import androidx.compose.material3.OutlinedTextField
 import androidx.compose.material3.Text
 import androidx.compose.material3.TextFieldDefaults
-import androidx.compose.material3.VerticalDivider
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
-import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
-import androidx.compose.runtime.getValue
 import androidx.compose.runtime.setValue
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.input.VisualTransformation
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import cinematic.watchAsState
 import kollections.firstOrNull
-import neat.Invalid
 import neat.Valid
 import neat.Validity
 import symphony.BaseField
 import symphony.toErrors
 
 @Composable
 fun InputTextField(
-    label: String,
-    value: String,
-    placeholder: String,
+    modifier: Modifier = Modifier,
```

Comment:
Modifier should be the first optional parameter in the parameter list; after all required parameters (except for trailing lambda parameters like card content) but before any other parameters with default values

Read here for more clarity https://blog.protein.tech/jetpack-compose-respect-the-contract-of-modifiers-ecbbe8ce03db